### PR TITLE
ENT-3125 Added Analytics pages with embedded Tableau

### PR DESCRIPTION
**Pull request by irfanuddinahmad**
Description: This PR adds a navigation in the sidebar and pages for embedded tableau

JIRA: https://openedx.atlassian.net/browse/ENT-3125

### CODE REVIEW

File: src/config/index.js [hunk 1]
```diff
@@ -23,11 +23,13 @@ const configuration = {
   USER_INFO_COOKIE_NAME: process.env.USER_INFO_COOKIE_NAME,
   NODE_ENV: process.env.NODE_ENV,
   CUSTOMER_SUPPORT_EMAIL: 'customersuccess@edx.org',
+  TABLEAU_URL: 'http://moconnell1453.sandbox.edx.org:8000/views/Superstore/Overview',
```
Comment (irfanuddinahmad) [inline src/config/index.js:26]: TODO: Move this to edx-internal

File: src/containers/EnterpriseApp/EnterpriseApp.test.jsx [hunk 2]
```diff
@@ -30,6 +30,8 @@ const initialState = {
     enterpriseId: 'test-enterprise-id',
     enableCodeManagementScreen: true,
     enableSubscriptionManagementScreen: true,
+    enableAnalyticsScreen: true,
+
```
Comment (zamanafzal) [inline src/containers/EnterpriseApp/EnterpriseApp.test.jsx:34]: I think this empty line has been added mistakenly.

**Review by zamanafzal — COMMENTED** [review]
LGTM 👍

**Review by zamanafzal — APPROVED** [review]

**Review by sameenfatima78 — APPROVED** [review]
LGTM!

**Pull request merged**